convert2rhel/utils.py

```diff
@@ -560,4 +560,17 @@ def backup(self):
             loggerinst.warning("Can't access %s" % TMP_DIR)
 
 
+def check_if_mount_is_readonly():
```

Comment:
```suggestion
def check_readonly_mounts():
    """Add a docstring that clearly describes what we're trying to achieve."""
```

The docstring can mention:

- Mounting directly to /mnt/ is not in line with Unix FS (https://en.wikipedia.org/wiki/Unix_filesystem).
- Having /mnt/ and /sys/ read-only causes the installation of the filesystem package to fail (https://bugzilla.redhat.com/show_bug.cgi?id=1887513, https://github.com/oamg/convert2rhel/issues/123).